gulpfile.babel.js

```diff
@@ -63,7 +63,7 @@ gulp.task('js', function() {
 
 /* Building CSS */
 gulp.task('css:main', function() {
-  return gulp.src('./wysadzulice/assets/styles/**/*.scss')
+  return gulp.src('./wysadzulice/assets/styles/**/main.scss')
```

Comment:
That's not a good idea to limit `scss` files only to one file. Leave here `*.scss`, please.